azure-samples/src/main/java/com/azure/management/samples/Utils.java

```diff
@@ -463,8 +463,8 @@ public static void print(Vault vault) {
                 .append("\n\tAccess policies: ");
         for (AccessPolicy accessPolicy : vault.accessPolicies()) {
             info.append("\n\t\tIdentity:").append(accessPolicy.objectId())
-                    .append("\n\t\tKey permissions: ").append(Joiner.on(", ").join(accessPolicy.permissions().getKeys()))
-                    .append("\n\t\tSecret permissions: ").append(Joiner.on(", ").join(accessPolicy.permissions().getSecrets()));
+                    .append("\n\t\tKey permissions: ").append(String.join(", ", accessPolicy.permissions().getKeys().stream().map(key -> key.toString()).collect(Collectors.toList())))
```

Comment:
There is a `Collectors.joining` as well.

Reply:
I'll test it later.

Reply:
No need to change now. Just a heads-up.

Reply:
I've added it for test, since it could shorten the code.